tests/bootstrap.php

```diff
@@ -1,17 +1,11 @@
 <?php
 
-declare(strict_types=1);
-
-use Doctrine\Common\Annotations\AnnotationReader;
 use Symfony\Component\Dotenv\Dotenv;
 
-require dirname(__DIR__) . '/vendor/autoload.php';
+require dirname(__DIR__).'/vendor/autoload.php';
 
-if (file_exists(dirname(__DIR__) . '/config/bootstrap.php')) {
-    require dirname(__DIR__) . '/config/bootstrap.php';
+if (file_exists(dirname(__DIR__).'/config/bootstrap.php')) {
+    require dirname(__DIR__).'/config/bootstrap.php';
 } elseif (method_exists(Dotenv::class, 'bootEnv')) {
-    (new Dotenv())->bootEnv(dirname(__DIR__) . '/.env');
+    (new Dotenv())->bootEnv(dirname(__DIR__).'/.env');
 }
-
-AnnotationReader::addGlobalIgnoredName('covers');
```

Comment:
I think these are needed until Doctrine Annotations 2 comes out. I added all these annotations to the library, but it was pushed into v2 (which is very very delayed).

Reply:
Oh, whoops. Never mind. Backported in https://github.com/doctrine/annotations/pull/331 all good. Remove away!